src/components/CvViewer/CvViewer.types.ts

```diff
@@ -8,38 +8,46 @@ export enum Template {
   ModernColumnBlack = 'modern column black'
 }
 
+export type PersonalInfoType = {
+  jobTitle?: string;
+  fullName?: string;
+  email?: string;
+  address?: string;
+  avatar?: string;
+  isShowAvatar?: boolean;
+  description?: string;
+};
+
+export type WorkExperienceType = {
+  id: string;
+  companyName?: string;
+  position?: string;
+  dateTime?: string[]; // american date format
+  startDate?: string; // american date format
+  endDate?: string; // american date format
```

Comment:
It's good that you made here one format because I have noticed that sometimes we have timestamp format when in other places we have American date format and it looked like some inconsistency

Reply:
right now we have american format everywhere